src/client/api/location/mod.rs

```diff
@@ -51,6 +51,13 @@ pub struct GeoCoordinates {
 pub enum GeoServiceVerified {
     Verified,
     Unverified,
+    #[serde(rename = "address_coordinates")]
+    AddressCoordinates,
+#[serde(rename_all = "snake_case")]
+pub enum GeoServiceVerified {
+    Verified,
+    Unverified,
+    AddressCoordinates,
 }
```

Comment:
```suggestion
#[serde(rename_all = "snake_case")]
pub enum GeoServiceVerified {
    Verified,
    Unverified,
    AddressCoordinates,
}
```

Doesn't look like the suggestions applied quite right to the PR

Reply:
Oh dear don't know how that happened.